CVLByExample/UnresolvedCallSummarization/README.md

```diff
@@ -8,12 +8,12 @@ This repository contains a smart contract named `TrusterLenderPool`, featuring a
 ## Usage
 
 ```cvl
-function _._ external => DISPATCH [
-    token.approve(address, uint256), 
+unresolved external in TrusterLenderPool.flashLoan(uint256,address,address,bytes) => DISPATCH(use_fallback=true) [
+    token.approve(address, uint256),
     currentContract._
 ] default NONDET;
 ```
-In this implementation, we summarize all unresolved calls into one of the following categories:
+In this implementation, we summarize all unresolved calls in the `flashloan` function into one of the following categories:
 - If the signature call matches the `approve` function, it is categorized as `token.approve(address, uint256)`.
 - If the signature call matches any of the functions within `currentContract`, their implementations are utilized.
 - If the callee contract has been resolved to `currentContract` but the signature did not match any of its functions, the fallback implementation will be used.
```

Comment:
do we have the link to the documentation ready ?
make sure the  link in the index table is updated

Reply:
Index Updated